package.json

```diff
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "@datadog/native-appsec": "2.0.0",
-    "@datadog/native-iast-rewriter": "1.0.0",
+    "@datadog/native-iast-rewriter": "1.1.2",
```

Comment:
Sorry, but we cannot jump from 1.0.0 to 1.1.2. I have to discuss a better solution with @uurien and @CarlesDD